### PR TITLE
Remove duplicated Novoda plugin config.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring

## :scroll: Description

Remove duplicated Novoda plugin config. 

Configuration has been moved from each module to parent `build.gradle.kts` into `subprojects` section.

## :bulb: Motivation and Context

https://github.com/getsentry/sentry-android/pull/537#discussion_r482122000

## :green_heart: How did you test it?

Build distribution packages `distZip` and compared with distributions created before the change.

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps

@marandaneto since I am not very fluent in Gradle I would appreciate your deeper review here.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -79,6 +81,30 @@ subprojects {
...
+
+        apply<ReleasePlugin>()
+
+        configure<PublishExtension> {
+            userOrg = Config.Sentry.userOrg
+            groupId = project.group.toString()
+            publishVersion = project.version.toString()
+            desc = Config.Sentry.description
+            website = Config.Sentry.website
+            repoName = Config.Sentry.javaRepoName
```
Comment (maciejwalkowiak) [inline build.gradle.kts:93]: Ok now I see the difference between Java and Android repo.

Comment (bruno-garcia) [reply build.gradle.kts]: What is the difference? We moved from https://github.com/getsentry/sentry-android into this one, isnt' it all the same now?

Comment (bruno-garcia) [reply build.gradle.kts]: Oh, is this a bintray repo?

Comment (bruno-garcia) [reply build.gradle.kts]: if so could it be named `bintrayRepo` please?

Comment (marandaneto) [reply build.gradle.kts]: no, java packages go under sentry-java, android packages go sentry-android and its not possible to change, we'd lose jcenter sync cus its a bintray limitation

**Comment by marandaneto** [comment]
@maciejwalkowiak I found problems doing something similar but on a few older Gradle's version , the kotlin build scripts were a bit limited yet, I think it should be fine now.
Just run `make dryRelease`, and you should be able to see the output of the release correctly.

**Review by marandaneto — APPROVED** [review]
thanks for doing this

**Comment by maciejwalkowiak** [comment]
Thanks @marandaneto. The output of `make dryRelease` looks good as far as i can tell.

File: build.gradle.kts (hunk 1 above)
Comment (bruno-garcia) [reply build.gradle.kts]: yeah I realize that on the second comment.
I suggested rename the variable. `repoName` is very ambiguous

**Review by bruno-garcia — APPROVED** [review]
SO happy to see this!

File: build.gradle.kts [hunk 2]
```diff
@@ -79,6 +81,30 @@ subprojects {
...
+
+        apply<ReleasePlugin>()
+
+        configure<PublishExtension> {
+            userOrg = Config.Sentry.userOrg
+            groupId = project.group.toString()
+            publishVersion = project.version.toString()
+            desc = Config.Sentry.description
+            website = Config.Sentry.website
+            repoName = if (project.name.contains("android")) Config.Sentry.androidRepoName else Config.Sentry.javaRepoName
```
Comment (bruno-garcia) [inline build.gradle.kts:93]: The suggestion is this, though PR is good to merge:

```suggestion
            repoName = if (project.name.contains("android")) Config.Sentry.androidBintrayRepoName else Config.Sentry.javaBintrayRepoName
```

Comment (marandaneto) [reply build.gradle.kts:93]: it makes easier to read but also verbose, its setting in the bintray plugin `repoName` field so it's kinda implicit that its bintray's repo name, I'm fine with this change though.
If you had a doubt about the var you are probably right, I wrote this part myself :D

**Pull request merged**